ci/Dockerfile

```diff
@@ -82,13 +82,11 @@ RUN apt-get install -y				\
 	netgen-headers				\
 	occt-draw				\
 	pybind11-dev				\
-	pyqt5-dev-tools				\
```

Comment:
This one is needed because it provides `pyuic`, `pyrcc` and `pylupdate` in Ubuntu.
There are problems getting them (at all, or at correct versions) through pyside packages.